book/chapters/chapter2/data_and_basic_modeling.qmd

```diff
@@ -8,33 +8,35 @@
+In the simplest definition, `r index('machine learning')` (ML) is the process of learning models of relationships from data.[Machine Learning/Supervised Learning]{.aside}
 `r index('Supervised learning')` is a subfield of ML in which datasets consist of labeled observations, which means that each data point consists of `r index('features')`, which are variables to make predictions from, and a `r index('target')`, which is the quantity that we are trying to predict.
 For example, predicting a car's miles per gallon (target) based on the car's properties (features) such as horsepower and the number of gears is a supervised learning problem, which we will return to several times in this book.
 In `mlr3`, we refer to datasets, and their associated metadata as `r index('tasks')` (@sec-tasks).
-The term 'tasks' is used to refer to the machine learning task (i.e., mathematical problem) that we are trying to solve.
+The term 'task' is used to refer to the prediction problem that we are trying to solve.
 Tasks are defined by the features used for prediction and the targets to predict, so there can be multiple tasks associated with any given dataset.
 For example, predicting miles per gallon (mpg) from horsepower is one task, predicting horsepower from mpg is another task, and predicting the number of gears from the car's model is yet another task.
 
 Supervised learning can be further divided into `r index('regression', aside = TRUE)` -- which is the prediction of numeric target values, e.g. predicting a car's mpg -- and `r index('classification', aside = TRUE)` -- which is the prediction of categorical values/labels, e.g., predicting a car's model.
-Other tasks are also encompassed by supervised learning, and we return to them in @sec-special, which also considers `r index('unsupervised learning')` tasks.
+@sec-special also discusses other tasks, such `r index('cost-sensitive classification', "cost-sensitive", parent = "classification")` or `r index('unsupervised learning')`.
 For any supervised learning task, the goal is to build a `r index('model')` that captures the relationship between the features and target, often with the goal of `r index('training', "model training")` the model to learn relationships about the data so it can make predictions for new and previously unseen data.
-A `r index('model', aside = TRUE)` is formally a mapping from a feature vector to predictions, such models are induced by passing `r index('training data')` to machine learning algorithms and include `r index('decision trees', "decision tree")`, `r index('support vector machines', "support vector machine")`, `r index('neural networks', "neural network")`, and many more.
+A `r index('model', aside = TRUE)` is formally a mapping from a feature vector to a prediction.
+A prediction can take many forms depending on the task; for classification, it can be a predicted label, or a set of predicted probabilities or scores.
```

Comment:
```suggestion
A prediction can take many forms depending on the task; for example, in classification this can be a predicted label, or a set of predicted probabilities or scores.
```